### ISSUE TITLE
Output text window don't auto scoll down

### ISSUE THREAD

**Issue by UmeSiyah** [sisoe24/Nuke-Tools-ST#1]
Hello,

I think it would be nice to have the `Output` section on the NukeServerSocket panel auto scroll down to the end of the text when a new output is receive.

**Comment by sisoe24** [comment]
Hi! Thanks for reaching out! I agree with you. I'm not sure if there is a way to update the console position. The code prints the output from the socket and does not interact directly with any console. If you have any suggestions or solutions in mind, please feel free to share them with me or, even better, you could open a PR.

**Comment by UmeSiyah** [comment]
Hello,

I will try to do it, but for the VSCode plugin. It'll easy to apply the update for Sublime Text ?

**Comment by sisoe24** [comment]
Hi! The VSCode API ([OutputChannel](https://code.visualstudio.com/api/references/vscode-api#OutputChannel)) doesn't offer a direct way to move the cursor to the bottom. But, the NukeTools extension has a feature that clears the previous output, making the result more visible. Also, vscode usually updates the console and automatically places the cursor at the bottom. When it doesn't, I usually clear the output manually by right-clicking and selecting "Clear Output." 

I am working on a major update for NukeServerSocket, including some breaking API changes. Once the update is released, I must also update the Sublime Text extension. I could check if Sublime Text has a feature to clear the output console before displaying new results to make it more similar to the one in VSCode.

vscode API for the output window:

**Comment by UmeSiyah** [comment]
Hi,

~~When I said moving the cursor down I was talking about inside Nuke. I tried something, I'll do a pull request soon :)~~
I'm sorry, it seems that the output section of the auto panel is already scrolling down...

I was thinking to working on the integration with Hiero. Should I wait you finish your update before starting?

**Comment by sisoe24** [comment]
I'm sorry for the confusion earlier. I thought you were referring to the output console of the Sublime Text extension instead of NukeServerSocket itself. I didn't think of the NukeServerSocket output console as a way to check the output but rather only as a logging event.

Although I have published a pre-release today, it might be best to wait for an official release. The Hiero feature can be difficult to implement due to the way Hiero loads itself. It might require a lot of work, and I'm unsure if it's worth it since NukeStudio can use Hiero and load NukeServerSocket. If you wish to try it out, I will happily help and answer every question.

**Comment by UmeSiyah** [comment]
I'll wait your official release to work on the integration with Hiero !
